resources/user-agents/browsers/slimerjs/slimerjs-general.json

```diff
@@ -5,16 +5,14 @@
   "standard": true,
```

Comment:
This file should be named `slimerjs-generic.json` no?

Reply:
done